### PR TITLE
Update get-all-vulnerabilities-by-machines.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated list of Odate filter queries

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5931

File: microsoft-365/security/defender-endpoint/get-all-vulnerabilities-by-machines.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ Retrieves a list of all the vulnerabilities affecting the organization per [mach
 
 - If the vulnerability has a fixing KB, it will appear in the response.
 - Supports [OData V4 queries](https://www.odata.org/documentation/).
-- The OData ```$filter``` is supported on all properties.
+- The OData's `$filter` query is supported on: `id`, `cveId`, `machineId`, `fixingKbId`, `productName`, `productVersion`, `severity` and `productVendor` properties.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-all-vulnerabilities-by-machines.md]:
```suggestion
- The OData's `$filter` query is supported on: `id`, `cveId`, `machineId`, `fixingKbId`, `productName`, `productVersion`, `severity`, and `productVendor` properties.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Pull request merged**